packages/i/injector/xmake.lua

```diff
@@ -0,0 +1,29 @@
+package("injector")
```

Comment:
`set_kind("library", {headeronly = true})`

Reply:
This package is using the compiled files from make/nmake so it would not be headeronly